p2p/config.go

```diff
@@ -28,7 +28,8 @@ const KeyFile = "node.key"
 // DefaultProtocolID ID
 const DefaultProtocolID = "/gossamer/dot/0"
 
-var DefaultBootnodes = []string{}
+// DefaultBootnodes holds the default bootnodes the client will always have
+var DefaultBootnodes []string
```

Comment:
this sets DefaultBootnodes to nil whereas before it was an empty array, might cause an issue?

Reply:
https://github.com/ChainSafe/gossamer/pull/585/files#diff-ee95e4ec6d917a2cfefc71b49e21a771R35

Reply:
good point, sorry for that. I updated with develop and used the value that was committed there.